string  name
float64  mov_time
geometry_msgs/Pose pose

